Tests.WebView/WebViewTestBase.cs

```diff
@@ -26,23 +27,20 @@ public class WebViewTestBase : TestBase<WebViewControl.WebView> {
         }
 
         protected Task Load(string html) {
-            var taskCompletionSource = new TaskCompletionSource<bool>();
+            var taskCompletionSource = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
Since we use task completion sources in the majority of tests, perhaps we should have our own representation of it (with the Option RunContinuationAsynchronously always set)

Reply:
only 3 occurrences use this flag